Integrations/integration-GRR.yml

```diff
@@ -34,8 +34,21 @@ script:
     var server = params.server + ':' + params.port;
     var insecure = params.insecure;
 
+    var getCSRFToken = function(url) {
+        var res = http(
+                url,
+                {
+                    Method: 'GET',
+                    Username: username,
+                    Password: password
+                },
+                insecure
+            );
+        return res.Cookies[0].Value;
```

Comment:
Is the CSRF token will always be the first in the cookies array? I would look for the correct key and then take the value (to be on the safe side).

Reply:
Complete